draft-irtf-cfrg-opaque.md

```diff
@@ -887,20 +895,21 @@ Input:
 - blind, an OPRF scalar value.
 - response, a RegistrationResponse structure.
 - server_identity, the optional encoded server identity.
-- client_identity, the optional encoded client identity.
+- client_identity, the encoded client identity.
 
 Output:
 - record, a RegistrationRecord structure.
 - export_key, an additional client key.
 
 Steps:
-1. y = Finalize(password, blind, response.data)
-2. randomized_pwd = Extract("", concat(y, Harden(y, params)))
-3. (envelope, client_public_key, masking_key, export_key) =
+1. info = client_identity
```

Comment:
This temp variable seems unnecessary -- why not just pass `client_idenity` to Finalize? (Same for the instance below, and Evaluate call above)

Reply:
Using client_id directly seems simplest here. Can we just do that?

Reply:
Good point @gtank ! Thanks :)